config/locales/server.en.yml

```
@@ -481,6 +481,7 @@ en:
   next_page: "next page →"
   prev_page: "← previous page"
   page_num: "Page %{num}"
+  crawler_content_hidden: "HTML content ommitted because you are logged in or using a modern mobile device."
```

```suggestion
  crawler_content_hidden: "HTML content omitted because you are logged in or using a modern mobile device."
```

I'd also be tempted to say "omitted from the crawler view...".